src/applications/check-in/components/PreCheckInAccordionBlock.jsx

```diff
@@ -96,7 +96,7 @@ const PreCheckInAccordionBlock = ({
         <strong>{title}</strong>
         {appointmentType === 'clinic' ? (
           <p>
-            t('a-staff-member-will-help-you-on-the-day-of-your-appointment')
```

Comment:
nice catch! was this just rendering out this literal text before?

Reply:
yup, @benbrasso-agile6 found it!